src/modules/kvs/lookup.c

```diff
@@ -606,16 +629,30 @@ bool lookup (lookup_t *lh)
                     lh->errnum = ENOTDIR;
                     goto done;
                 }
-                reftmp = json_string_value (vp);
+                if (!(reftmp = treeobj_get_blobref (lh->wdirent, 0))) {
```

Comment:
is use useful to check that the number of blobrefs is always one?  Might be handy later when we make changes to allow more than one.